.scripts/ci/push_bench_results.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#The script takes output.txt, removes every line that doesn't have "test"
+#in it and pushes benchmark result to Victoria Metrics
+#Benchmark name should have underscores in the name instead of spaces (e.g. async/http_concurrent_round_trip/8)
```

Comment:
```suggestion
# The script takes output.txt, removes every line that doesn't have "test"
# in it and pushes benchmark result to Victoria Metrics
# Benchmark name should have underscores in the name instead of spaces (e.g. async/http_concurrent_round_trip/8)
```